cmd/clusterctl/client/cluster/objectgraph_test.go

```diff
@@ -1131,7 +1131,7 @@ func TestObjectGraph_DiscoveryByNamespace(t *testing.T) {
 				objs: func() []runtime.Object {
 					objs := []runtime.Object{}
 					objs = append(objs, test.NewFakeCluster("ns1", "cluster1").Objs()...)
-					objs = append(objs, test.NewFakeCluster("ns2", "cluster1").Objs()...)
+					objs = append(objs, test.NewFakeClusterWithForceLabel("ns2", "cluster1").Objs()...)
```

Comment:
I think we should add a dedicated test case instead of conflating into an existing one.
Also, for sake of completeness, we should add a test case in moveTests, so we test that the overall move process take forceMove into account

Reply:
ok, will create a new one